android/app/src/main/java/com/mattermost/rnbeta/CustomPushNotification.java

```diff
@@ -263,10 +263,12 @@ protected void postNotification(int id, Notification notification) {
                 .setGroupSummary(true)
                 .setStyle(messagingStyle)
                 .setSmallIcon(smallIconResId)
-                .setNumber(Integer.parseInt(badge))
                 .setVisibility(Notification.VISIBILITY_PRIVATE)
                 .setPriority(Notification.PRIORITY_HIGH)
                 .setAutoCancel(true);
+        if (badge != null) {
```

Comment:
We already have a check for badge != null at 232, maybe place this there instead?

Also this fixes the crash but it still does not show the right message in the notification.